### ISSUE TITLE
offsetY and offsetX in Validations are useless?

### ISSUE THREAD

**Issue by escudero89** [mercadolibre/chico#1318]
Could it be that those values are not affecting the target at all? I tried looking up in the source code, but I couldn't see a place where those values are actually being used.

I could be incredible wrong though, so I apologize myself in advance.

Greetings.

**Comment by battaglr** [comment]
@escudero89, which version are you using?

**Comment by escudero89** [comment]
Chico `2.0.3`, and the looking up of the source code was donde over the `master` branch.

**Comment by battaglr** [comment]
@escudero89, if I'm not mistaken the way in which we handle the validation messages was modified in v2x. Let's invite @atma to the conversation. :grinning:

**Comment by atma** [comment]
@escudero89 `Validation` is extends the `Bubble` which extends the `Popover` that uses `Positioner` for its positioning, so offsetY and offsetX are passing to that component trough this complex chain. Do you have some troubles with the positioning of validator's messages?

**Comment by escudero89** [comment]
To be honest, I wasn't so sure what was `offsetY` actually changing, due that we use a version of Chico Mobile in which the validations doesn't have bubbles. At least for me the [docs](http://atma.github.io/chico/api-doc/current/ui/ch.validation/) are quite confusing about what this `target` is in the description of `offset(X/Y)`.

I tried to understand how the `offset` was being used, and since I didn't notice a change in our validations nor I'd found how it was being used in `master`, I reported this issue.

It can be closed now.
